msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -142,39 +141,41 @@ In the **Authorized client applications** section, ensure all of the following `
 
 ## Sideload your app in Teams
 
-The final step is to sideload your updated personal tab ([app package](/microsoftteams/platform/concepts/build-and-test/apps-package)) in Microsoft Teams. Once completed, your app will be available to run in Office and Outlook, in addition to Teams.
+The final step to running your app in Office and Outlook is to sideload your updated personal tab [app package](..//concepts/build-and-test/apps-package.md) in Microsoft Teams.
```

Comment:
Looks like the link starts with "..//", which doesn't seem right...although it seems to work in the build, you might want to check it's correct.